client/src/css/Home.scss

```diff
@@ -1 +1,52 @@
 @import '../styles/colors.scss';
+
+@font-face {
+    font-family: 'Tommy';
+    src: local('Tommy'), url('../fonts/Tommy.ttf') format('truetype');
+  }
+
+
+
+.sessionCard {
+  box-shadow: none !important;
+  text-align: left !important;
+  width: 400px !important;
+  height: 100px !important;
+  margin-bottom: 20px !important;
+  border-radius: 15px !important;
+  font-family: Tommy !important;
+}
+
+.ui.card>.content>.header {
+  font-family: Tommy !important;
+}
+
+.homeHeader {
+  margin-top: 50px;
+  margin-left: 100px;
+  margin-bottom: 50px;
+  font-size: 80px;
+  text-align: left;
+  font-family: Tommy;
+}
+
+h1 {
+  font-family: Tommy;
+}
+
+.insideCard {
+  margin-top: 5px !important;
+  padding: 20px !important;
+  font-family: Tommy !important;
+}
+
+.btn {
+  margin-left: 300px;
+  background-color: #c4c4c4;
+  color: black;
+  padding: 5px 15px;
+  border-radius: 5px;
+}
+.btn:hover {
+  color: black;
+}
```

Comment:
Please give this a more specific className since there will be multiple buttons in our app. For example, `join-btn` in this case. If the Join button will have the same styling as other buttons, use a general className like `general-btn`, but move the code to the general `styles.scss` folder since it'll be used in more than one place.